tools/chapel-py/chapel.cpp

```diff
@@ -696,284 +546,24 @@ struct PerNodeInfo {
   };
 };
 
-#define METHOD_TABLE(TAG, ...)\
+#define METHOD_TABLE_BEGIN(TAG) \
```

Comment:
I'm surprised you can define the `METHOD_TABLE_BEGIN` macro here despite it being defined by `method-tables.h`.

Reply:
It only gets defined in `method-tables.h` if it is not already defined before the file is included